test/functional/notes_controller_test.rb

```diff
@@ -692,6 +692,15 @@ def teardown
     actual = notes.first
     assert expected == actual.created
   end
+     
+  test 'first three posts in /liked should be sorted by likes' do
+    get :liked
+    # gets first notes
+    notes = assigns(:notes)[0...3]
+     # sort_by is from lowest to highest so it needs to be reversed
+    assert notes.sort_by { |note| note.cached_likes }.reverse ==  notes
```

Comment:
LGTM. 🎉 
Please add the 2 tests which you added previously too.
Please submit the task for approval.
Thanks

Reply:
Thanks! 😄 
Uh the other two tests are already in the pull request. I submitted for review on the GCI dashboard.